cmd/catalog.go

```diff
@@ -19,6 +20,9 @@ Example:
 
 	# Add a catalog and specify the branch to use
 	$ rancher catalog add --branch awesomebranch foo https://my.catalog
+
+	# Add a catalog and specify the helm version to use
```

Comment:
We should probably specify the two choices are v2 and v3

Reply:
Yea or they might use `helmV3` or `3`. Otherwise lgtm.

Reply:
or try using a version that has a minor or patch